### ISSUE TITLE
How to delete quotes when tag message has Multiple lines

### ISSUE THREAD

**Issue by miyouzi** [ericcornelissen/git-tag-annotation-action#23]
Hi, thank you for your project, it's helpful ! And I have a little question.

My action yml below:

```yaml

......

- name: Get tag message
      id: tag_data
      uses: ericcornelissen/git-tag-annotation-action@v1

......

- name: Create Release
      id: create_release
      uses: actions/create-release@v1
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      with:
        tag_name: ${{ env.RELEASE_VERSION }}
        release_name: "${{ env.RELEASE_TITLE }}"
        body: |
          ${{ steps.tag_data.outputs.git-tag-annotation }}
```

Then I found the release body has quotes at the beginning and ending. How to delete it ?

![XBSJ(KN$51N(TSI)DEW61LK](https://user-images.githubusercontent.com/24269465/127107376-a4dc8050-c703-43ba-9feb-8c085b1a809f.png)

The whole yml file is at https://github.com/miyouzi/aniGamerPlus/blob/master/.github/workflows/Release-build.yml

**Comment by ericcornelissen** [comment]
Thank you for opening an issue @miyouzi

Comparing your example to [a workflow of one of my own projects](https://github.com/ericcornelissen/shescape/blob/c6b1c07aa847bcb79bb47749dd69ca83896183ab/.github/workflows/release.yml#L36-L49) that is nearly identical and where [release](https://github.com/ericcornelissen/shescape/releases) are created correctly (*), my initial though is that changing 

```
        body: |
          ${{ steps.tag_data.outputs.git-tag-annotation }}
```

to 

```
        body: ${{ steps.tag_data.outputs.git-tag-annotation }}
```

may solve the problem.

Also, I noticed that [the output](https://github.com/miyouzi/aniGamerPlus/runs/3168768413?check_suite_focus=true#step:6:24) of [the `Test ENV` step](https://github.com/miyouzi/aniGamerPlus/blob/c006982c5bd4d70af19abdc293f5a592137dda71/.github/workflows/Release-build.yml#L35-L44) shows outputs the `git-tag-annotation` without quotes - suggesting that it's a problem with the release action as well.

If you need a fix fast, you can give the above a try. If that doesn't work, you could compare these two workflows yourself and look for a possible culprit. Else, I will have time to test this later this week.

---

(*): note that it's been a while since the last release, theoretically the issue could've been introduced since then. E.g. in #19 or #22.

**Comment by miyouzi** [comment]
Thank you for your reply.

I have tried to put ` ${{ steps.tag_data.outputs.git-tag-annotation }}` on the same line as `body`, but didn't work.
The action output is [here](https://github.com/miyouzi/aniGamerPlus/runs/3177657040)

![image](https://user-images.githubusercontent.com/24269465/127251238-976d6e78-e0dc-4d61-94d9-5b1a732d8fd1.png)

I'll try to use another create-release action, and wait for your test.

**Comment by ericcornelissen** [comment]
Thank for trying that for me @miyouzi :+1:

Quick update: I tested a little bit more and the culprit seems to be the usage of `runs-on: windows-2019` (and probably other windows version as well). This definitely seems to be a valid bug. However, I did not yet have a chance to see what [the command this action uses](https://github.com/ericcornelissen/git-tag-annotation-action/blob/415fbdab535ba295b826a08d175b9b656a336978/src/main.js#L15) normally outputs on Windows so I can't say for sure how to fix it yet. Anyway, I will try to figure that out as soon as possible and release a fix.

**Comment by ericcornelissen** [comment]
I have been able to confirm that this is indeed the expected behavior on Windows due to the use of single quotes in [`--format='%(content)'`](https://github.com/ericcornelissen/git-tag-annotation-action/blob/415fbdab535ba295b826a08d175b9b656a336978/src/main.js#L15) (*). I'm already working on a fix. Unfortunately just removing the quotes from the format results in problems on Unix systems, but I think I have found a solution nonetheless.


---


(*): Note that running, e.g. `git for-each-ref --format='%(content)' refs/tags/v1.0.0` may not include quotes depending on the shell you're using. However, if you use [Node's child_process](https://nodejs.org/api/child_process.html) it seems to be always included (I tested only with `exec`, `execSync`, and `execFileSync`) due to the default shell used.
